src/frontend/utils/conversejs/converse.ts

```diff
@@ -10,47 +11,52 @@ import { nicknameManagementPlugin } from './converse-plugins/nicknameManagementP
 import { participantsTrackingPlugin } from './converse-plugins/participantsTrackingPlugin';
 
 export const initConverse = (xmpp: XMPP) => {
-  chatPlugin.addPlugin(xmpp);
-  logoutPlugin.addPlugin();
-  marshaJoinDiscussionPlugin.addPlugin(xmpp);
-  nicknameManagementPlugin.addPlugin(xmpp);
-  participantsTrackingPlugin.addPlugin();
+  const { isChatInitialized, setIsChatInitialized } =
+    useChatItemState.getState();
```

Comment:
I'm not really fan of this way to determine if the chat has alredy been mounted or not. There is absolutely no need to use an external store for that, This can be managed internally in the function

Reply:
I changed it to managed initialisation internally